File: 1491-average-salary-excluding-the-minimum-and-maximum-salary/1491-average-salary-excluding-the-minimum-and-maximum-salary.c
double average(int* salary, int salarySize){
int max,min,i,sum;
    sum=max=min=salary[0];
    for(i=1;i<salarySize;i++){
        if(max<salary[i])
            max=salary[i];
        if(min>salary[i])
            min=salary[i];
        sum+=salary[i];
    }
    return (sum-min-max)/(salarySize-2.0);    
}

File: 1491-average-salary-excluding-the-minimum-and-maximum-salary/1491-average-salary-excluding-the-minimum-and-maximum-salary.py
class Solution:
    def average(self, salary: List[int]) -> float:
        sum=min=max=salary[0]
        for i in range(1,len(salary)):
            if (salary[i]>max):
                max=salary[i]
            if (salary[i]<min):
                min=salary[i]
            sum+=salary[i]
            
        return (sum-min-max)/(len(salary)-2)
        